### DISCUSSION TITLE
Kimai 2 plugins (importer and backup)

### DISCUSSION THREAD

**Discussion by hyssop** [kimai/Paid-plugins#72] [Questions & Answers] [answered]
RE: Importer
Is it free or not? It says free on the plugin page, and also says purchase. (I'm pretty sure when I recently looked at it it was downloadable, for free. Now the free link sends me to a payment screen.)

RE: Easy Backup
On the plugin page it say not compatible with v2, but the download and github page says it now is compatible with v2. Please clarify.

**Comment by kevinpapst** [comment, marked as answer]
Importer is a pay-what-you-want plugin. It was always.

Easy Backup is not my plugin. Please ask the author. If the info in the marketplace is outdated, then it needs an update.

**Reply by hyssop** [reply]
Oh, duh... sorry I missed that. :) Thanks, I'll try before I buy.

I have posted also there, but brought it here because I saw that it said v2 compatible on Kimai Marketplace.
